pkg/v22/errors/error.go

```diff
@@ -0,0 +1,73 @@
+package errors
+
+import (
+	"regexp"
+	"strings"
+
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	chartConfigNotInstalledErrorText = "the server could not find the requested resource (get chartconfigs.core.giantswarm.io)"
+)
+
+var (
```

Comment:
Approach is based on https://github.com/giantswarm/errors/blob/2640113be13f2eec85fe785ed74418a1a2852e5d/tenant/error.go#L10.